### PR TITLE
[ID] Add Autopilot, Cinema, and Co-op mod articles

### PR DESCRIPTION
**Pull request by hikikuman**
Add Indonesian translation for [Autopilot](https://osu.ppy.sh/wiki/en/Game_modifier/Autopilot), [Cinema](https://osu.ppy.sh/wiki/en/Game_modifier/Cinema), and [Co-op](https://osu.ppy.sh/wiki/en/Game_modifier/Co-op) articles (game modifier).

### CODE REVIEW

File: wiki/Game_modifier/Autopilot/id.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Deskripsi
+
+*Perhatian: Penggunaan mod Auto akan menyebabkan permainan menjadi unranked dan tidak akan tersimpan di papan peringkat lokal.*
+
+Mod **Autopilot** adalah sebuah [game modifier](/wiki/Game_modifier) untuk [osu!standard](/wiki/Game_mode/osu!) yang akan mengontrol pergerakan kursor dan secara sistematis berpindah tepat ke tengah [hit object](/wiki/Hit_object) berikutnya, dan menyerahkan tapping kepada pemain. Saat bermain dengan menggunakan mod Autopilot, pemain tidak akan bisa menggerakkan kursor mereka, kecuali mereka sedang berada di layar jeda (pause).
+
+Mod Autopilot juga menonaktifkan antarmuka in-game, sehingga [skor](/wiki/Score), [kombo](/wiki/Beatmapping/Combo), dan miss menjadi tidak terlihat, meskipun hal tersebut dapat diaktifkan kembali dengan menekan tombol `Shift`+`Tab`.
+
+Selain itu, nilai spins per minute (SPM) yang dilakukan oleh mod Autopilit juga sama seperti spins per minute yang dilakukan oleh mod [Spun Out](/wiki/Game_modifier/Spun_out). Dengan demikian, pemain tidak dapat meningkatkan SPM dengan cara memutar spinner menggunakan kursor mereka.
```
Comment (ZeroPyrozen) [inline wiki/Game_modifier/Autopilot/id.md:35]: Ada typo di `Autopilit` seharusnya `Autopilot`

File: wiki/Game_modifier/Cinema/id.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Deskripsi
+
+*Perhatian: Penggunaan mod Auto akan menyebabkan permainan menjadi unranked.*
+
+Mod **Cinema** adalah sebuah [game modifier](/wiki/Game_modifier) yang bertujuan agar pemain dapat bersantai dan menonton video atau storyboard dari sebuah beatmap tanpa munculnya [hit object](/wiki/Hit_object). Mod ini sama seperti mod [Auto](/wiki/Game_modifier/Auto), tetapi **hanya background dan storyboard yang ditampilkan**.
+
+Karena mod Cinema merupakan variasi dari mod Auto, hitsound yang berasal dari hit object akan tetap terdengar. Namun elemen tayangan ulang dan layar hasil akan dinonaktifkan bersamaan dengan dim background yang disetel ke 0%.
+
+![Gameplay Auto vs. Cinema](img/GM_Cinema.jpg)
```
Comment (ZeroPyrozen) [inline wiki/Game_modifier/Cinema/id.md:33]: Baris ini sudah berubah dari `Gameplay Auto vs. Cinema` menjadi `Auto vs. Cinema gameplay vs the Cinema mod (right)")`. Sumber:  [[EN] Cinema (mod)](https://github.com/ppy/osu-wiki/blob/master/wiki/Game_modifier/Cinema/en.md)

File: wiki/Game_modifier/Co-op/id.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- Deskripsi: `Dua kali jumlah key-nya, dua kali keseruannya!`
+- Mode permainan yang cocok: ![][o!m]
+
+## Deskripsi
+
+*Perhatian: Penggunaan mod Co-op akan menyebabkan permainan menjadi unranked.*
+
+Mod **Co-op** adalah sebuah [game modifier](/wiki/Game_modifier) untuk [osu!mania](/wiki/Game_mode/osu!mania) yang memperkenalkan arena permainan kedua dengan set kontrol kedua berdasarkan jumlah *x*K pada saat itu (7K -> 7K + 7K = 14K). Untuk [beatmap](/wiki/Beatmaps) spesifik osu!mania, *x*K dibagi menjadi setengah (7K -> 4K + 3K = 7K).
+
+![Perbandingan gameplay Co-op](img/GM_COOP.jpg "Perbandingan antara permainan biasa (atas-kiri) dengan permainan co-op (atas-kanan) dari beatmap spesifik osu!mania, dan permainan biasa (bawah-kiri) dengan permainan co-op (bawah-kanan) dari beatmap non-spesifik osu!mania")
```
Comment (ZeroPyrozen) [inline wiki/Game_modifier/Co-op/id.md:34]: Baris ini sudah berubah menjadi 

```
![osu!mania-specific` Co-op gameplay comparison](img/CP-comparison-mania-specific-beatmap.jpg "Comparison between a normal play (left) vs. a play with the Co-op mod enabled (right) on an osu!mania-specific beatmap")

![Converted osu!mania beatmap Co-op gameplay comparison](img/CP-comparison-converted-beatmap.jpg "Comparison between a normal play (left) vs. a play with the Co-op mod enabled (right) on a converted osu!mania beatmap")
```
Sumber: [[EN] Co-op (mod)](https://github.com/ppy/osu-wiki/blob/master/wiki/Game_modifier/Co-op/en.md)

**Review by ZeroPyrozen — CHANGES_REQUESTED** [review]
Mohon diubah terlebih dahulu baris yang sudah diberikan comment.

File: wiki/Game_modifier/Autopilot/id.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+- Singkatan: AP
+- Jenis: Spesial
+- Pengali Skor: 0.00x
+- Tombol shortcut: `X`
+- Deskripsi: `Kursor akan digerakkan secara otomatis - Anda hanya perlu mengikuti ritme lagu! ** UNRANKED **`
+- Mode permainan yang cocok: ![][o!s]
+
+## Deskripsi
+
+*Perhatian: Penggunaan mod Auto akan menyebabkan permainan menjadi unranked dan tidak akan tersimpan di papan peringkat lokal.*
```
Comment (AncuL001) [inline wiki/Game_modifier/Autopilot/id.md:29]:
```suggestion
*Perhatian: Penggunaan mod Auto akan menyebabkan permainan menjadi unranked dan tidak tersimpan di papan peringkat lokal.*
```

File: wiki/Game_modifier/Autopilot/id.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+Mod **Autopilot** adalah sebuah [game modifier](/wiki/Game_modifier) untuk [osu!standard](/wiki/Game_mode/osu!) yang akan mengontrol pergerakan kursor dan secara sistematis berpindah tepat ke tengah [hit object](/wiki/Hit_object) berikutnya, dan menyerahkan tapping kepada pemain. Saat bermain dengan menggunakan mod Autopilot, pemain tidak akan bisa menggerakkan kursor mereka, kecuali mereka sedang berada di layar jeda (pause).
```
Comment (AncuL001) [inline wiki/Game_modifier/Autopilot/id.md:31]:
```suggestion
Mod **Autopilot** adalah sebuah [game modifier](/wiki/Game_modifier) untuk [osu!standard](/wiki/Game_mode/osu!) yang akan mengontrol pergerakan kursor dan secara sistematis berpindah tepat ke tengah [hit object](/wiki/Hit_object) berikutnya sehingga menyerahkan tapping kepada pemain. Saat bermain menggunakan mod Autopilot, pemain tidak akan bisa menggerakkan kursor mereka, kecuali ketika berada di layar jeda (pause).
```

File: wiki/Game_modifier/Autopilot/id.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+Mod Autopilot juga menonaktifkan antarmuka in-game, sehingga [skor](/wiki/Score), [kombo](/wiki/Beatmapping/Combo), dan miss menjadi tidak terlihat, meskipun hal tersebut dapat diaktifkan kembali dengan menekan tombol `Shift`+`Tab`.
```
Comment (AncuL001) [inline wiki/Game_modifier/Autopilot/id.md:33]:
```suggestion
Mod Autopilot juga menonaktifkan antarmuka in-game sehingga [skor](/wiki/Score), [kombo](/wiki/Beatmapping/Combo), dan miss menjadi tidak terlihat, meskipun hal tersebut dapat diaktifkan kembali dengan menekan tombol `Shift`+`Tab`.
```

File: wiki/Game_modifier/Cinema/id.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+- Singkatan: CM
+- Jenis: Spesial
+- Pengali Skor: 1.00x
+- Tombol shortcut: `V` `V` atau `Shift`+`V` (atau tahan `Ctrl`+`Shift` saat sedang memilih sebuah [beatmap](/wiki/Beatmaps))
+- Mode permainan yang cocok: ![][o!s] ![][o!t] ![][o!c] ![][o!m]
+
+## Deskripsi
+
+*Perhatian: Penggunaan mod Auto akan menyebabkan permainan menjadi unranked.*
```
Comment (AncuL001) [inline wiki/Game_modifier/Cinema/id.md:27]:
```suggestion
*Perhatian: Penggunaan mod Cinema akan menyebabkan permainan menjadi unranked.*
```

File: wiki/Game_modifier/Cinema/id.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+Mod **Cinema** adalah sebuah [game modifier](/wiki/Game_modifier) yang bertujuan agar pemain dapat bersantai dan menonton video atau storyboard dari sebuah beatmap tanpa munculnya [hit object](/wiki/Hit_object). Mod ini sama seperti mod [Auto](/wiki/Game_modifier/Auto), tetapi **hanya background dan storyboard yang ditampilkan**.
+
+Karena mod Cinema merupakan variasi dari mod Auto, hitsound yang berasal dari hit object akan tetap terdengar. Namun elemen tayangan ulang dan layar hasil akan dinonaktifkan bersamaan dengan dim background yang disetel ke 0%.
```
Comment (AncuL001) [inline wiki/Game_modifier/Cinema/id.md:31]:
```suggestion
Karena mod Cinema merupakan variasi dari mod Auto, hitsound yang berasal dari hit object akan tetap terdengar. Namun, elemen tayangan ulang dan layar hasil akan dinonaktifkan bersamaan dengan dim background yang disetel ke 0%.
```

**Review by AncuL001 — CHANGES_REQUESTED** [review]
don't forget to fix pyrozen's issues

File: wiki/Game_modifier/Autopilot/id.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+*Berbeda konteks dengan [Auto (mod)](/wiki/Game_modifier/Auto).*
+
+## Tentang
+
+- Singkatan: AP
+- Jenis: Spesial
+- Pengali Skor: 0.00x
+- Tombol shortcut: `X`
+- Deskripsi: `Kursor akan digerakkan secara otomatis - Anda hanya perlu mengikuti ritme lagu! ** UNRANKED **`
+- Mode permainan yang cocok: ![][o!s]
```
Comment (DoryanR) [inline wiki/Game_modifier/Autopilot/id.md:25]:
```suggestion
- Mode permainan yang cocok: ![][osu!]
```

File: wiki/Game_modifier/Autopilot/id.md (hunk 5 above)
Comment (DoryanR) [inline wiki/Game_modifier/Autopilot/id.md:31]:
```suggestion
Mod **Autopilot** adalah sebuah [game modifier](/wiki/Game_modifier) untuk [osu!](/wiki/Game_mode/osu!) yang akan mengontrol pergerakan kursor dan secara sistematis berpindah tepat ke tengah [hit object](/wiki/Hit_object) berikutnya, dan menyerahkan tapping kepada pemain. Saat bermain dengan menggunakan mod Autopilot, pemain tidak akan bisa menggerakkan kursor mereka, kecuali mereka sedang berada di layar jeda (pause).
```

File: wiki/Game_modifier/Autopilot/id.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+*Perhatian: Penggunaan mod Auto akan menyebabkan permainan menjadi unranked dan tidak akan tersimpan di papan peringkat lokal.*
+
+Mod **Autopilot** adalah sebuah [game modifier](/wiki/Game_modifier) untuk [osu!standard](/wiki/Game_mode/osu!) yang akan mengontrol pergerakan kursor dan secara sistematis berpindah tepat ke tengah [hit object](/wiki/Hit_object) berikutnya, dan menyerahkan tapping kepada pemain. Saat bermain dengan menggunakan mod Autopilot, pemain tidak akan bisa menggerakkan kursor mereka, kecuali mereka sedang berada di layar jeda (pause).
+
+Mod Autopilot juga menonaktifkan antarmuka in-game, sehingga [skor](/wiki/Score), [kombo](/wiki/Beatmapping/Combo), dan miss menjadi tidak terlihat, meskipun hal tersebut dapat diaktifkan kembali dengan menekan tombol `Shift`+`Tab`.
+
+Selain itu, nilai spins per minute (SPM) yang dilakukan oleh mod Autopilit juga sama seperti spins per minute yang dilakukan oleh mod [Spun Out](/wiki/Game_modifier/Spun_out). Dengan demikian, pemain tidak dapat meningkatkan SPM dengan cara memutar spinner menggunakan kursor mereka.
+
+[o!s]: /wiki/shared/mode/osu.png "osu!standard"
```
Comment (DoryanR) [inline wiki/Game_modifier/Autopilot/id.md:37]:
```suggestion
[osu!]: /wiki/shared/mode/osu.png "osu!"
```

File: wiki/Game_modifier/Cinema/id.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*\
+*Berbeda konteks dengan [Cinematography](https://en.wikipedia.org/wiki/Cinematography "Wikipedia").*
+
+## Tentang
+
+- Singkatan: CM
+- Jenis: Spesial
+- Pengali Skor: 1.00x
+- Tombol shortcut: `V` `V` atau `Shift`+`V` (atau tahan `Ctrl`+`Shift` saat sedang memilih sebuah [beatmap](/wiki/Beatmaps))
+- Mode permainan yang cocok: ![][o!s] ![][o!t] ![][o!c] ![][o!m]
```
Comment (DoryanR) [inline wiki/Game_modifier/Cinema/id.md:23]:
```suggestion
- Mode permainan yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
```

File: wiki/Game_modifier/Cinema/id.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+
+Mod **Cinema** adalah sebuah [game modifier](/wiki/Game_modifier) yang bertujuan agar pemain dapat bersantai dan menonton video atau storyboard dari sebuah beatmap tanpa munculnya [hit object](/wiki/Hit_object). Mod ini sama seperti mod [Auto](/wiki/Game_modifier/Auto), tetapi **hanya background dan storyboard yang ditampilkan**.
+
+Karena mod Cinema merupakan variasi dari mod Auto, hitsound yang berasal dari hit object akan tetap terdengar. Namun elemen tayangan ulang dan layar hasil akan dinonaktifkan bersamaan dengan dim background yang disetel ke 0%.
+
+![Gameplay Auto vs. Cinema](img/GM_Cinema.jpg)
+
+Mod ini memiliki efek yang sama di semua mode permainan.
+
+[o!s]: /wiki/shared/mode/osu.png "osu!standard"
```
Comment (DoryanR) [inline wiki/Game_modifier/Cinema/id.md:37]:
```suggestion
[osu!]: /wiki/shared/mode/osu.png "osu!"
```

File: wiki/Game_modifier/Cinema/id.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+[o!t]: /wiki/shared/mode/taiko.png "osu!taiko"
```
Comment (DoryanR) [inline wiki/Game_modifier/Cinema/id.md:38]:
```suggestion
[osu!taiko]: /wiki/shared/mode/taiko.png "osu!taiko"
```

File: wiki/Game_modifier/Cinema/id.md [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+[o!c]: /wiki/shared/mode/catch.png "osu!catch"
```
Comment (DoryanR) [inline wiki/Game_modifier/Cinema/id.md:39]:
```suggestion
[osu!catch]: /wiki/shared/mode/catch.png "osu!catch"
```

File: wiki/Game_modifier/Cinema/id.md [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+[o!m]: /wiki/shared/mode/mania.png "osu!mania"
```
Comment (DoryanR) [inline wiki/Game_modifier/Cinema/id.md:40]:
```suggestion
[osu!mania]: /wiki/shared/mode/mania.png "osu!mania"
```

File: wiki/Game_modifier/Co-op/id.md [hunk 16]
```diff
@@ -0,0 +1,55 @@
...
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*\
+*Berbeda konteks dengan [Multiplayer](/wiki/Multi).*
+
+## Tentang
+
+- Singkatan: CP atau CO
+- Jenis: Spesial
+- Pengali Skor: 0.70x–1.00x (lihat [tabel dibawah](#score-multiplier))
+- Deskripsi: `Dua kali jumlah key-nya, dua kali keseruannya!`
+- Mode permainan yang cocok: ![][o!m]
```
Comment (DoryanR) [inline wiki/Game_modifier/Co-op/id.md:26]:
```suggestion
- Mode permainan yang cocok: ![][osu!mania]
```

File: wiki/Game_modifier/Co-op/id.md [hunk 17]
```diff
@@ -0,0 +1,55 @@
...
+
+*Perhatian: Penggunaan mod Co-op akan menyebabkan permainan menjadi unranked.*
+
+Mod **Co-op** adalah sebuah [game modifier](/wiki/Game_modifier) untuk [osu!mania](/wiki/Game_mode/osu!mania) yang memperkenalkan arena permainan kedua dengan set kontrol kedua berdasarkan jumlah *x*K pada saat itu (7K -> 7K + 7K = 14K). Untuk [beatmap](/wiki/Beatmaps) spesifik osu!mania, *x*K dibagi menjadi setengah (7K -> 4K + 3K = 7K).
+
+![Perbandingan gameplay Co-op](img/GM_COOP.jpg "Perbandingan antara permainan biasa (atas-kiri) dengan permainan co-op (atas-kanan) dari beatmap spesifik osu!mania, dan permainan biasa (bawah-kiri) dengan permainan co-op (bawah-kanan) dari beatmap non-spesifik osu!mania")
+
+### Pengali skor
+
+Tabel berikut hanya mempengaruhi beatmap convert [osu!standard](/wiki/Game_mode) yang dimainkan pada mode permainan osu!mania dengan menggunakan mod Co-op:
```
Comment (DoryanR) [inline wiki/Game_modifier/Co-op/id.md:38]:
```suggestion
Tabel berikut hanya mempengaruhi beatmap convert [osu!](/wiki/Game_mode) yang dimainkan pada mode permainan osu!mania dengan menggunakan mod Co-op:
```

File: wiki/Game_modifier/Co-op/id.md [hunk 18]
```diff
@@ -0,0 +1,55 @@
...
+
+*Catatan: Mod Co-op tidak memberikan penalti pada beatmap spesifik osu!mania.**
+
+Penjelasan mendetail tentang arena permainan kedua dapat ditemukan di [halaman osu!mania bagian "Co-op"](/wiki/Game_mode/osu!mania#co-op).
+
+### Trivia
+
+- Mod Co-op diperkenalkan pada 5 Maret 2015
+
+[o!m]: /wiki/shared/mode/mania.png "osu!mania"
```
Comment (DoryanR) [inline wiki/Game_modifier/Co-op/id.md:55]:
```suggestion
[osu!mania]: /wiki/shared/mode/mania.png "osu!mania"
```

**Review by DoryanR — CHANGES_REQUESTED** [review]
In line with the new standards

File: wiki/Game_modifier/Cinema/id.md (hunk 2 above)
Comment (hikikuman) [reply wiki/Game_modifier/Cinema/id.md:33]: on my way

File: wiki/Game_modifier/Autopilot/id.md [hunk 19]
```diff
@@ -0,0 +1,37 @@
...
+
+## Deskripsi
+
+*Catatan: Penggunaan mod Auto akan menyebabkan permainan menjadi unranked dan tidak tersimpan di papan peringkat lokal.*
+
+Mod **Autopilot** adalah sebuah [game modifier](/wiki/Game_modifier) untuk [osu!](/wiki/Game_mode/osu!) yang akan mengontrol pergerakan kursor dan secara sistematis berpindah tepat ke tengah [hit object](/wiki/Hit_object) berikutnya, dan menyerahkan tapping kepada pemain. Saat bermain dengan menggunakan mod Autopilot, pemain tidak akan bisa menggerakkan kursor mereka, kecuali mereka sedang berada di layar jeda (pause).
+
+Mod Autopilot juga menonaktifkan antarmuka in-game sehingga [skor](/wiki/Score), [kombo](/wiki/Beatmapping/Combo), dan miss menjadi tidak terlihat, meskipun hal tersebut dapat diaktifkan kembali dengan menekan tombol `Shift`+`Tab`.
+
+Selain itu, nilai spins per minute (SPM) yang dilakukan oleh mod Autopilot juga sama seperti spins per minute yang dilakukan oleh mod [Spun Out](/wiki/Game_modifier/Spun_out). Dengan demikian, pemain tidak dapat meningkatkan SPM dengan cara memutar spinner menggunakan kursor mereka.
```
Comment (wowcake) [inline wiki/Game_modifier/Autopilot/id.md:35]:
```suggestion
Selain itu, nilai spins per minute (SPM) yang dilakukan oleh mod Autopilot juga sama seperti spin per minute yang dilakukan oleh mod [Spun Out](/wiki/Game_modifier/Spun_out). Dengan demikian, pemain tidak dapat meningkatkan SPM dengan cara memutar spinner menggunakan kursor mereka.
```

**Review by wowcake — CHANGES_REQUESTED** [review]
should be good to be merged after this

**Review by wowcake — APPROVED** [review]
Rechecked, good to go

**Review by TicClick — APPROVED** [review]

**Pull request merged**